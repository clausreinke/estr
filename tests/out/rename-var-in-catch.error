Error: Command failed: WARNING! hoisting var declaration over catch of same name: e
{ start: { line: 20, column: 8 },
  end: { line: 20, column: 16 } }
WARNING! hoisting function declaration over catch of same name: f
{ start: { line: 28, column: 4 },
  end: { line: 28, column: 24 } }
cannot rename declaration hoisted over catch
e { line: 20, column: 8 }
