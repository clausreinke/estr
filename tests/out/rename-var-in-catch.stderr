WARNING! hoisting var declaration over catch of same name:  e
{ start: { line: 20, column: 8 },
  end: { line: 20, column: 16 } }
WARNING! hoisting function declaration over catch of same name:  f
{ start: { line: 28, column: 4 },
  end: { line: 28, column: 24 } }
cannot rename declaration hoisted over catch
{ type: 'Identifier',
  name: 'e',
  range: [ 471, 472 ],
  loc: { start: { line: 20, column: 8 }, end: { line: 20, column: 9 } },
  hoistConflict: true,
  occurrences: 
   [ { type: 'Identifier',
       name: 'e',
       range: [Object],
       loc: [Object],
       innerScopes: [] } ],
  innerScopes: [] }
